saleor/checkout/views/__init__.py

```diff
@@ -138,30 +138,29 @@ def checkout_index(request, checkout):
             variant=line.variant,
             initial=initial,
             discounts=discounts,
-            taxes=taxes,
         )
+        total_line = tax_interface.calculate_checkout_line_total(line, discounts)
+        variant_price = total_line / line.quantity
```

Comment:
Not sure about this. If you leave this as-is, the unit price may have tens of decimal places. If you quantize it, it's likely that `variant_price * line.quantity != total_line`.

Reply:
What do you propose? From avalara docs:  `to determine the price-per-each, divide the amount value by the quantity value.` So I have the only `total_line` and somehow need to get `unit_price`.